tests/test_charm.py

```diff
@@ -122,20 +127,25 @@ def test_charm_setup_gitlab_by_id_and_secret(self, _set_legend_creds_mock, _mock
 
         # Check relations data:
         creds = self._get_gitlab_creds_from_config(
-            config_values, cert=_mock_ssl.PEM_cert_to_DER_cert.return_value
+            config_values, cert=mock_sslsock.getpeercert.return_value
```

Comment:
instead of ``cert=mock_sslsock.getpeercert.return_value``, you can write ``cert=cert``, since you're already setting ``        mock_sslsock.getpeercert.return_value = cert`` above.

Reply:
Updated it to be equal to `test_cert_der` everywhere for consistency.